amd/src/ggbq.js

```diff
@@ -14,13 +14,24 @@ define(['jquery', '//www.geogebra.org/apps/deployggb.js'], function ($, GGBApple
      * Created by Christoph on 25.08.19.
      */
 
+    const scalingContainers = {};
     let resizeTimeout;
     /**
      * Resizes the ggb scaling containers to make the ggb applet scale properly to fit into its container.
      */
     const resizeScalingContainer = () => {
         clearTimeout(resizeTimeout);
-        resizeTimeout = setTimeout(() => $(".ggbscalingcontainer").width($(".qtext").parent().parent().width(), 250));
+        resizeTimeout = setTimeout(
+            () => Object.values(scalingContainers).forEach((containerClass) => {
+                    // We need to use getElementsByClassName because colons are not allowed for jquery and Vanilla JS querySelector.
+                    const scalingContainer = document.getElementsByClassName(containerClass)[0];
+                    // We retrieve the formulation div container, because this gives us the correct width to adapt
+                    // the scaling container to.
+                    const formulationDivStyle = window.getComputedStyle(
```

Comment:
I would call the constant `qtextDivStyle`.

Reply:
Well, it's not the qtext div, but the formulation div, see `parentElement.parentElement` in line 31

Reply:
Sorry, you're right. :+1: